src/views/Proposals/index.tsx

```diff
@@ -78,68 +80,86 @@ function Proposals() {
   const desktop = useMediaQuery(theme.breakpoints.up(1048));
   const mobile = useMediaQuery(theme.breakpoints.down(705));
   let navigate = useNavigate();
-  const [rows, setRows] = useState<IRow[]>([])
-  const handleClickMenu = (link: string, proposal_name: string) => {
-    if(link === "/proposaldetails"){
+  const [rows, setRows] = useState<IRow[]>([]);
+  const handleClickMenu = (link: string, proposal: string | number) => {
+    if (link === "/proposaldetails") {
       navigate(link, {
         state: {
-          key : proposal_name,
+          key: proposal,
         },
         replace: true,
       });
-    } else {  
+    } else {
       navigate(link);
     }
   };
 
-  useEffect(()=> {
+  function calculateDaysAndHours(dateTimeStr: string) {
+    const targetDateTime = new Date(dateTimeStr);
```

Comment:
I would suggest adding 
`dateTimeStr = dateTimeStr + 'Z'`
as it currently tries to read the date using the user's timezone instead of UTC.